CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [Unreleased]
+
+## Breaking Changes
+
+This release contains an additional peer dependency on `expo-secure-store`. Run the bugsnag-expo-cli to add this dependency to your project:
```

Comment:
I've followed the approach we used here for this: https://github.com/bugsnag/bugsnag-expo/blob/v50/next/CHANGELOG.md#v4400-2022-04-19

but is it better off in an `UPGRADING.md`?

Reply:
I like the sound of an upgrade guide and keeping code snippets out of the changelog 👍🏻